src/gui/mainwindow.cpp

```diff
@@ -1913,6 +1913,7 @@ void MainWindow::handleUpdateCheckFinished(ProgramUpdater *updater, const bool i
         msgBox->setAttribute(Qt::WA_DeleteOnClose);
         msgBox->setAttribute(Qt::WA_ShowWithoutActivating);
         msgBox->setDefaultButton(QMessageBox::Yes);
+        msgBox->setWindowModality(Qt::NonModal);
```

Comment:
Seems this contradicts with our latest findings here. 169c4991d545b15b12589a6ae11b4bfad1035c37
You should revert this line and use `msgBox->show()` at line 1925 (only this line).

Reply:
> You should revert this line

But isn't the point of this PR precisely to make the dialog non-modal?

Reply:
msgBox->show() is non-modal, isn't?

Reply:
I tested it now, this line should stay and changing to `msgBox->show()` should be done, otherwise it won't work.